src/VisualStudio/Core/Def/Implementation/ChangeSignature/ChangeSignatureDialog.xaml.cs

```diff
@@ -153,9 +171,14 @@ private void Add_Click(object sender, RoutedEventArgs e)
                     addParameterViewModel.TypeBinds);
 
                 _viewModel.AddParameter(addedParameter);
-            }
 
-            SetFocusToSelectedRow();
+                SetFocusToSelectedRow(true);
+            }
+            else
+            {
+                // Cancel/esc was pressed.
+                AddButton.Focus();
```

Comment:
I think we just want to let the focus stay where it naturally was, especially since the user can `Alt+a` to launch the Add Parameter dialog from anywhere and they might not expect focus to move to a button somewhere just because they used the shortcut. What do you think?